butterknife/src/main/java/butterknife/internal/SupportV4Utils.java

```diff
@@ -0,0 +1,45 @@
+package butterknife.internal;
+
+import android.content.res.Resources;
+import android.graphics.drawable.Drawable;
+import android.support.v4.graphics.drawable.DrawableCompat;
+import android.util.TypedValue;
+
+import static butterknife.internal.CompatUtils.getColor;
+
+/** Requires v4 Support Library. Check {@link #HAS_SUPPORT_V4_LIBRARY} before use. */
+final class SupportV4Utils {
+  static final boolean HAS_SUPPORT_V4_LIBRARY = hasSupportV4LibraryOnClasspath();
+  static final TypedValue OUT_VALUE = new TypedValue();
```

Comment:
Yeah this is fine. We don't have a strong precedent on mutable "constant" naming anyway.